nat/nat.go

```diff
@@ -87,6 +87,15 @@ func validateProto(proto string) bool {
 	return false
 }
 
+func ValidatePort(port string) error {
+	_, portStr := SplitProtoPort(port)
+	if _, error := ParsePort(portStr); error != nil {
```

Comment:
I think it should be:
```
_, err := ParsePort(portStr)
return err
```
Also, `error` is keyword for type in golang, better not use it for your variables.